common-aggregator/pom.xml

```diff
@@ -1,63 +0,0 @@
-<?xml version="1.0"?>
-<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd"
-         xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-    <modelVersion>4.0.0</modelVersion>
-
-    <artifactId>notifications-common-aggregator</artifactId>
```

Comment:
What is the benefit of removing the `common-aggregator` module? Now, several classes including `AggregationCommand` are included in the `backend` module classpath while they're not needed.

Reply:
Soon Aggregator will use Events instead of EmailAggregation. So we can move all Event mapping classes related to `common-aggregator`, or we can remove `common-aggregator` module to depend of common directly. The second option seems more reasonable to me. WDYT?